cmd/cluster.go

```diff
@@ -922,6 +960,30 @@ func updateCluster(args []string) error {
 	}
 	cur.Kubernetes = k8s
 
+	auditFlags := &models.V1Audit{}
```

Comment:
As for updates you allow this to be `nil`, I think it's cleaner when you do `var auditFlags *models.V1Audit{}` and initialize the var if `viper.IsSet("audit")`.